cfgov/v1/tests/test_wagtail_hooks.py

```diff
@@ -208,8 +208,8 @@ def test_tag_param_returns_resources_tagged_that_way(self):
 
 @unittest.skipIf(wagtail.VERSION >= (2, 0), "No need to test in Wagtail 2+")
 class TestWhitelistOverride(SimpleTestCase):
-    # Borrowed from https://github.com/wagtail/wagtail/blob/v1.13.4/wagtail
-    # /wagtailcore/tests/test_dbwhitelister.py
+    # Borrowed from https://github.com/wagtail/wagtail/blob/master/wagtail
+    # /core/tests/test_whitelist.py
 
     def test_whitelist_hooks(self):
         """Test that DbWhitelister does not strip new elements and attributes.
```

Comment:
```suggestion
        """Test that Whitelister does not strip new elements and attributes.
```

Minor note, but the class dropped the "Db" prefix.